libs/domain/picking/offline/data-plugin.ts

```diff
@@ -16,16 +16,14 @@ import {
 import { PickingListEntity, PickingProductEntity } from './entities';
 import { PickingListOnlineAdapter } from './services';
 
-export class OfflineDataPlugin extends ExecPlugin {
+export class OfflineDataPlugin implements AppPlugin {
   protected subscription?: Subscription;
 
-  constructor() {
-    super(() => undefined);
+  getName(): string {
+    return 'oryx$pickingOfflineData';
```

Comment:
We've dropped `$` sign to use `.` instead:
```suggestion
    return 'oryx.pickingOfflineData';
```